### PR TITLE
remove the unnecessary fmt.Scanln command

### LINKED ISSUE #59: Investigate whether Scanln can be removed
There is this bit in the test runner code: https://github.com/exercism/go-test-runner/blob/main/testrunner/extract.go#L34

It means that if you run the test runner outside of the docker container with the command from the README `go run . testrunner/testdata/practice/passing outdir` then you need to press enter a lot of times to actually get the output json file. We don't yet understand why it works correctly in the production setting. We want to find out how this works to decide whether the Scanln part can be removed or guarded by some flag to activate it only if needed.

### PR DESCRIPTION
**Pull request by mhmmdd**
Fixes #59  in the way suggested here https://github.com/exercism/go-test-runner/issues/59#issuecomment-999874166

Without this code test runner has been tested following scenarios and it seems perfectly fine.

## 1. Running the test runner on Windows/Linux
```bash
$ go run . testrunner/testdata/practice/passing test
```
## 2. Running the test runner in the container by passing in the slug name
```bash
# Build the local docker image
$ docker build --rm -t exercism/test-runner .

# Run the test runner and see the results in the test folder
$ docker run --network none -v $(pwd)/testrunner/testdata/practice/gigasecond:/solution -v $(pwd)/test:/test exercism/test-runner gigasecond /solution /test
```
## 3. Running the test runner in the container by using attached mode
```bash
# Build the local docker image
$ docker build --rm -t exercism/test-runner .

# Run a container with attachmed mode
$ docker run --entrypoint /bin/sh --rm --name exercism-go-test-runner -d --network none -v $(pwd):/opt/test-runner -v $(pwd)/test:/test -it exercism/test-runner

# Access the container
$ docker exec -it --user appuser $(docker ps -q --filter name=exercism-go-test-runner) /bin/sh

# Run the test runner inside the container
$ go run . testrunner/testdata/practice/passing /test
```

### CODE REVIEW

**Review by andrerfcsantos — APPROVED** [review]

**Review by junedev — APPROVED** [review]

**Comment by junedev** [comment]
@mhmmdd FYI: We need approval from one of the two admins to merge this change. Because of the holiday season, it can take a while until one of them has time to approve this.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
